docs/website/docs/deployment-configurations/cpu-dylib.md

```diff
@@ -131,11 +131,11 @@ concrete values.
 
 <!-- TODO(??): troubleshooting -->
 
-[android-cc]: /building-from-source/android/
-[get-started]: /building-from-source/getting-started/
+[android-cc]: ../building-from-source/android.md
```

Comment:
Generally: use relative links to source files within markdown files and site generators can figure out the right mapping from those to permalinks. Absolute paths assume things about the url structure of the published site that might not be true (e.g. `https://google.github.io/iree/` vs `https://google.github.io/`, path ending in `/` vs `.html`).